api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -24,6 +24,27 @@ message KubernetesObjectData {
   oneof kind_specific {
     KubernetesPodData pod = 100;
     KubernetesNodeData node = 101;
+    KubernetesServiceData service = 102;
+  }
+}
+
+message KubernetesServiceData {
+  // List of IP addresses associated with the Kubernetes service
+  repeated string ip_addresses = 1;
+  // List of service ports details
+  repeated ServicePorts ports = 2;
+  // Type of Kubernetes service (e.g., ClusterIP, NodePort, LoadBalancer, etc.)
+  string service_type = 3;
```

Comment:
Remove the `service_` prefix, it's redundant with the message name, and just using `type` matches the name of the field in the k8s `Service` spec.
I don't think `type` is a reserved keyword??
```suggestion
  string type = 3;
```